deltacat/compute/compactor/model/compact_partition_params.py

```diff
@@ -413,6 +413,14 @@ def num_rounds(self) -> int:
     def num_rounds(self, num_rounds: int) -> None:
         self["num_rounds"] = num_rounds
 
+    @property
+    def parquet_to_pyarrow_inflation(self) -> int:
```

Comment:
Wouldn't a float be more suitable here if we want to use non-integers?

Reply:
Addressed in new PR.